### PR TITLE
Clipboard and Notes

### PR DESCRIPTION
**Pull request by shogenapps**
Keep useful things for pasting in the clipboard and useful notes in the notes.

### CODE REVIEW

File: src/features/register_feature_options.js [hunk 1]
```diff
@@ -130,6 +130,13 @@ registerFeature({
   category: "Editing",
 });
 
+registerFeature({
+  name: "Clipboard and Notes",
+  id: "clipboardAndNotes",
+  description: "Keep useful things for pasting in the clipboard and useful notes in the notes.",
+  category: "Global",
+});
```
Comment (RobPavey) [inline src/features/register_feature_options.js:138]: When my change is merged you will want to set the defaultValue here

File: src/features/randomProfile/randomProfile.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
 import $ from "jquery";
 import { getRandomProfile } from "../../core/common";
-import { getPerson } from "wikitree-js";
```
Comment (RobPavey) [inline src/features/randomProfile/randomProfile.js:3]: Is this a conflict with PR https://github.com/wikitree/wikitree-browser-extension/pull/119 ?

Comment (shogenapps) [reply src/features/randomProfile/randomProfile.js:3]: No, I don't think so.  I found I needed that in the common.js file for getRandomProfile, which I moved to the common.js file to give the My Menu feature access to it.  I hope that makes sense, and I hope the getPerson call is in common.js now (I don't have time to check).  I'll be back later.

File: src/features/register_feature_options.js (hunk 1 above)
Comment (shogenapps) [reply src/features/register_feature_options.js:138]: OK. Thanks.

**Pull request merged**